### PR TITLE
Shaya hirani khayam

### PR DESCRIPTION
**Pull request by shayaHirani**
change hafez and add khaiyam

### CODE REVIEW

File: hafz [hunk 1]
```diff
@@ -1,4 +1,5 @@
-hafz 1
-hafe 2
-hafez3
-hafez4
+hafez 1
+hafez 2
+hafez 3
```
Comment (shayaHirani) [inline hafz:3]: remove hafez 3

File: khaiyam [hunk 2]
```diff
@@ -0,0 +1,2 @@
+khaiyam 1
+khaiyam 2
```
Comment (shayaHirani) [inline khaiyam:2]: add hafez

**Review by shayaHirani — COMMENTED** [review]
they are good changes

**Comment by shanaHirani** [comment]
add hafez khiaym to hafez

**Pull request closed without merging**